docs/src/index.md

```diff
@@ -4,9 +4,27 @@ CurrentModule = GIFImages
 
 # GIFImages
 
-Documentation for [GIFImages](https://github.com/ashwani-rathee/GIFImages.jl).
+This is the documentation for [GIFImages](https://github.com/ashwani-rathee/GIFImages.jl).
 
-```@index
+GIFImages.jl provides support for decoding and encoding GIF images.
+
+# Usage 
+For decoding purposes, GIFImages.jl currently supports `gif_decode` which 
+decode the GIF image as colorant matrix. The source data needs to be a filename.
+
+#### Arguments
+- `filepath::AbstractString` : Path to the gif file
+- `use_localpalette::Bool=true` : While decoding, using this argument use of local color map or global color map can be specified
```

Comment:
Actually, it's unclear to me what "local"/"global" color map means at the first glance. But I figure there's a (global) colormap for the GIF file, and a (local) colormap for each slice of the GIF file?

Reply:
Yes, there is a global(common for all) and local color map(some slices) which tell what color palettes to use. I will update it to make it more clearer.